iterator.go

```diff
@@ -44,7 +44,7 @@ import (
 // internalIterator contains 7 valid positions and 2 exhausted positions.
 //
 // Iterator positioning methods must set iterPos to iterPosCur{Foward,Backward}
-// iff the user key at the current internalIterator position equals the
```

Comment:
this is actually not a typo. `iff` is short-hand for "[if and only if](https://en.wikipedia.org/wiki/If_and_only_if)"

Reply:
Ah, thank you!